packages/kit/src/views/Setting/pages/Tab/SubSettingsPage.tsx

```
@@ -18,34 +18,37 @@ import type { RouteProp } from '@react-navigation/native';
-  const configList = useMemo(() => {
+  const config = useMemo(() => {
     return settingsConfig
-      ? settingsConfig
-          ?.find((item) => item?.title === name)
-          ?.configs.filter((item) => item && item.length)
-      : [];
+      ? settingsConfig?.find((item) => item?.name === name)
+      : null;
   }, [name, settingsConfig]);
+  const configList = useMemo(() => {
+    return config?.configs.filter((item) => item && item.length) || [];
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)